### PR TITLE
python-package: add graphviz.Digraph parameters

### PR DESCRIPTION
**Pull request by henry0312**
also add a example with graphviz :)

### CODE REVIEW

**Comment by guolinke** [comment]
@wxchan can you help to review this ?

File: examples/python-guide/plot_example.py [hunk 1]
```diff
@@ -53,3 +53,7 @@
 print('Plot 84th tree...')  # one tree use categorical feature to split
 ax = lgb.plot_tree(gbm, tree_index=83, figsize=(20, 8), show_info=['split_gain'])
 plt.show()
+
+print('Plot 85th tree with graphviz...')
```
Comment (wxchan) [inline examples/python-guide/plot_example.py]: is it 85th?

Comment (henry0312) [inline examples/python-guide/plot_example.py]: No, it's 84th.
I'll fix.

**Comment by henry0312** [comment]
updated!

**Pull request merged**